packages/typespec-test/test/healthInsights_radiologyinsights/generated/typespec-ts/README.md

```diff
@@ -1,46 +1,74 @@
-# AzureHealthInsights REST client library for JavaScript
+# AzureHealthInsights client library for JavaScript
 
-Azure AI Health Insights provides an API that serves insight models, specific for Health & Life Sciences, that perform analysis and provide inferences to be used by a human.
-
-**Please rely heavily on our [REST client docs](https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/rest-clients.md) to use this library**
+This package contains an isomorphic SDK (runs both in Node.js and in browsers) for AzureHealthInsights client.
 
-Key links:
+Azure AI Health Insights provides an API that serves insight models, specific for Health & Life Sciences, that perform analysis and provide inferences to be used by a human.
 
-- [Package (NPM)](https://www.npmjs.com/package/@azure-rest/health-insights-radiologyinsights)
-- [API reference documentation](https://docs.microsoft.com/javascript/api/@azure-rest/health-insights-radiologyinsights?view=azure-node-preview)
+[Package (NPM)](https://www.npmjs.com/package/@azure-rest/health-insights-radiologyinsights) |
+[API reference documentation](https://docs.microsoft.com/javascript/api/@azure-rest/health-insights-radiologyinsights?view=azure-node-preview) |
```

Comment:
Seems there is an additional '|' in the end

Reply:
created issue for it https://github.com/Azure/autorest.typescript/issues/2883